docs/data/pages.ts

```diff
@@ -1,8 +1,15 @@
 const pages = [
   {
-    pathname: '/x/advanced-components',
-    title: 'Overview',
+    pathname: '/x/getting-started/overview',
+    scopePathnames: ['/x/getting-started/'],
+    title: 'Getting started',
     icon: 'DescriptionIcon',
+    children: [
+      { pathname: `/x/getting-started/overview` },
```

Comment:
Taking as example the pages under "Data Grid", the Overview is located at https://deploy-preview-4692--material-ui-x.netlify.app/x/react-data-grid/, not `/x/react-data-grid/overview`. If we keep `/x/getting-started/overview`, it means that accessing `/x/getting-started/` won't take the user to a broader page, but to a 404. It breaks a bit the semantic of the URLs.

In implementation terms, `docs/pages/x/getting-started/overview.js` should be renamed to `docs/pages/x/getting-started/index.js` to be accessible as `/x/getting-started`.